torch/_inductor/kernel/mm.py

```diff
@@ -284,13 +290,17 @@ def tuned_addmm(inp, mat1, mat2, *, alpha=1, beta=1, layout=None):
             )
 
     if static_shape and is_nonzero and use_cutlass_template(layout, m, n, k):
-        CUTLASSGemmTemplate.add_cutlass_gemm_choices(
-            choices,
-            layout,
-            [mat1, mat2, inp_expanded],
-            alpha=alpha,
-            beta=beta,
-        )
+        if (
+            WrapperCodeGen.statically_known_int_or_none(inp_expanded.layout.stride[-1])
+            != 0
+        ):
```

Comment:
is this only an issue for this particular cutlass gemm? what about the others?

Reply:
I don't know why, but I have seen CUDA illegal memory access errors on recent versions of Cutlass if this dimension had a stride of zero. It's a rare case and most likely a bug in some Cutlass Kernel(s). It's only relevant for the bias in addmm, so yes, the others don't have this issue.

Reply:
Added an explanatory comment.